packages/@aws-cdk/aws-events-targets/test/ecs/event-rule-target.test.ts

```diff
@@ -58,6 +58,249 @@ test('Can use EC2 taskdef as EventRule target', () => {
   });
 });
 
+test('Throws error for lacking of taskRole ' +
```

Comment:
Open suggestion for how to DRY.

Tried `beforeEach` + a function consolidating GIVEN and THEN part of the tests, but using `beforeEach` made the tests correlate to each other, caused failures like naming collision (`There is already a Construct with name 'TaskDef' in Stack`).

Reply:
That should not happen if you also create a new `stack` in the `beforeEach()`.

```ts
let stack: cdk.Stack;
let vpc: ec2.Vpc;
let cluster: ec2.Cluster;
beforeEach(() => {
  stack = new cdk.Stack();
  vpc = new ec2.Vpc(stack, 'Vpc', { maxAzs: 1 });
  cluster = new ecs.Cluster(stack, 'EcsCluster', { vpc });
  cluster.addCapacity('DefaultAutoScalingGroup', {
    instanceType: new ec2.InstanceType('t2.micro'),
  });
});
```

Feels like an appropriate setup suite.